app/presenters/details_presenter.rb

```diff
@@ -0,0 +1,47 @@
+require "govspeak"
+
+module Presenters
+  class DetailsPresenter
+    attr_reader :content_item_details
+
+    def initialize(content_item_details)
+      @content_item_details = SymbolizeJSON.symbolize(content_item_details)
+    end
+
+    def details
+      @_details ||=
+        begin
+          content_item_details.inject({}) do |seed, hsh|
```

Comment:
you could use `each_with_object` to avoid having to reset "seed" at the end. Also you can split key, val in the argument. `content_item_details.each_with_object({}) {|(key,val), seed| ... }`